src/index.ts

```diff
@@ -936,9 +938,9 @@ export default class Purchases {
 
   /**
    * Enables/Disables debugs logs
-   * @param {Boolean} enabled Enable or not debug logs
+   * @param {boolean} enabled Enable or not debug logs
    */
-  public static setDebugLogsEnabled(enabled: boolean): Promise<void> {
+  public static setDebugLogsEnabled(enabled: boolean): void {
```

Comment:
Based on earlier clarifications and the return type update, I suspect the below should not use `return` any longer 😄

Reply:
oops, I missed that one, thanks!